scripts/prepare-crds

```diff
@@ -74,10 +78,10 @@ fi
 copyAnnotations=(catalog.cattle.io/release-name catalog.cattle.io/certified catalog.cattle.io/experimental catalog.cattle.io/namespace)
 for a in ${copyAnnotations[@]}; do
 	v=$(yq r ${f}/charts/Chart.yaml "annotations[${a}]")
-	if [[ ${a} == "catalog.cattle.io/release-name" ]]; then
-		v="${v}-crd"
-	fi
 	if ! [[ -z ${v} ]]; then
+		if [[ ${a} == "catalog.cattle.io/release-name" ]]; then
+			v="${v}-crd"
+		fi
 		yq w -i ${f}/charts-crd/Chart.yaml "annotations[${a}]" "${v}"
 	fi
 done
```

Comment:
@aiyengar2 would be good for you to review whole PR, but especially this part ^

Reply:
Good catch, this would be a bug if the annotation was not set as it would have resulted in:

```yaml
annotations:
    catalog.cattle.io/release-name: "-crd"
```

Looks good to me, thanks for the change!